docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-19_remove-sample-data-dir.txt

```diff
@@ -0,0 +1 @@
+* The deprecated configuration variable ```SAMPLE_DATA_DIR``` has been removed in favour of user installation of the ```iris_sample_data``` package.
```

Comment:
When installing via Conda, the package is called 'iris-sample-data', which is also the name of the GitHub repo.